client/src/pages/Location.js

```diff
@@ -30,12 +36,24 @@ const Tour = () => {
         </p>
       </Card>
       <ImageCard>
-        <img src={placeholder} alt="placeholder" />
-        <img src={placeholder} alt="placeholder" />
-        <img src={placeholder} alt="placeholder" />
-        <img src={placeholder} alt="placeholder" />
-        <img src={placeholder} alt="placeholder" />
-        <img src={placeholder} alt="placeholder" />
+        <Link to="/Pictures">
```

Comment:
usually urls are written with small letters
```suggestion
        <Link to="/pictures">
```